src/jobs/cancel-older-awaiting-approvals.yml

```diff
@@ -3,7 +3,7 @@ description: |
   Depends on jq being present.
 parameters:
   circle_token:
-    default: ${CIRCLECI_TOKEN}
+    default: '${CIRCLECI_TOKEN}'
```

Comment:
Are these quotes necessary?  Was this just due to linting?

Reply:
Yeah this was linting

Reply:
It would be good to mention that in the PR description too.  I noticed the mention of reformatting for indentation, but I had to assume this was for linting.  The more you can avoid assumptions, the easier it is to review and also avoid questions or potential issues.

Another approach is to break up PRs like this into the pure stylistic/reformatting changes in one PR.  That way, it's easy to review.  No semantic changes are expected.  It makes 2 much smaller PRs that are easier to reason about and easier to describe, than one larger PR that does multiple things.

Reply:
True, Ill split up the big linter changes